packages/dashboard/src/components/DashboardGridLayout/AddCardPanel/AddCardPanel.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+
+import { Panel, PanelType } from 'office-ui-fabric-react/lib/Panel';
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+
+import { AddCard } from './AddCard/AddCard';
+import { getStyles } from './AddCardPanel.styles';
+import { IAddCardPanelProps, IAddCardPanelState, IAddCardPanelStyles } from './AddCardPanel.types';
+import { IDGLCard } from '../../../index';
+
+export class AddCardPanel extends React.Component<IAddCardPanelProps, IAddCardPanelState> {
+  constructor(props: IAddCardPanelProps) {
+    super(props);
+    this.state = {
+      isOpen: false,
+      flyoutStyle: {}
+    };
+  }
+
+  public render(): JSX.Element {
+    const { header, isOpen, cards } = this.props;
+    return (
+      <>
+        <Panel
+          onRenderBody={this._renderAddCardItems.bind(this, cards, header)}
+          type={PanelType.custom}
+          customWidth={'480px'}
+          isOpen={isOpen}
+          isBlocking={false}
+          onDismiss={this._onDismiss}
+          // tslint:disable-next-line:jsx-ban-props
+          style={this.state.flyoutStyle}
+        />
+      </>
+    );
+  }
+
+  private _onDismiss = () => {
+    this.setState({ isOpen: false });
+  };
+
+  private _renderAddCardItems = (addCardItems: IDGLCard[], header: string): JSX.Element => {
+    const getClassNames = classNamesFunction<{}, IAddCardPanelStyles>();
+    const classNames = getClassNames(getStyles!);
+    const addCardItemsList: JSX.Element[] = [];
+    addCardItems.map((addCardItem: IDGLCard, index: number) => {
+      addCardItemsList.push(
+        <div key={index}>
+          <AddCard
+            title={
+              addCardItem.addCardInfo
+                ? addCardItem.addCardInfo.addCardPanelHeader
+                  ? addCardItem.addCardInfo.addCardPanelHeader
+                  : 'Default Title'
+                : 'Default title'
+            }
+            description={
+              addCardItem.addCardInfo
+                ? addCardItem.addCardInfo.addCardPanelBodyText
+                  ? addCardItem.addCardInfo.addCardPanelBodyText
+                  : 'Default Description'
+                : 'Default Description'
```

Comment:
>Default Description' [](start = 19, length = 20)

have empty string. we don't want to put any default values

Reply:
Changed to empty string